src/go/pt-k8s-debug-collector/dumper/dumper.go

```diff
@@ -58,6 +54,14 @@ func New(location, namespace, resource string) Dumper {
 	}
 	if len(resource) > 0 {
 		resources = append(resources, resource)
+
+		if resource == "pxc" {
+			resources = append(resources,
+				"perconaxtradbbackups",
+				"perconaxtradbclusterbackups",
+				"perconaxtradbclusterrestores",
+				"perconaxtradbclusters")
+		}
```

Comment:
can we also improve psmdb case?
```suggestion
		} else if resource == "psmdb" {
			resources = append(resources,
				"perconaservermongodbbackups",
				"perconaservermongodbrestores",
				"perconaservermongodbs")
		}
```

Reply:
I added it in 010c9796a8566039013b9a27d9fc7dcc73e33faf

Also, I think we should use `resourceType` function here from #500
https://github.com/percona/percona-toolkit/blob/c8b821219638c4b819116731dc45592a81556af1/src/go/pt-k8s-debug-collector/dumper/dumper.go#L357